src/RenderToBuffer.ts

```diff
@@ -22,14 +22,15 @@ export default class RenderToBuffer {
   public mesh: Mesh;
   public camera: OrthographicCamera;
 
-  constructor(fragmentSrc: string, uniforms: { [key: string]: IUniform }) {
+  constructor(fragmentSrc: string, uniforms: { [key: string]: IUniform }, transparent = false) {
     this.scene = new Scene();
     this.geometry = new PlaneGeometry(2, 2);
 
     this.material = new ShaderMaterial({
       vertexShader: renderToBufferVertShader,
       fragmentShader: fragmentSrc,
       uniforms,
+      transparent,
```

Comment:
Allow some render passes to be transparent, so they can overwrite the existing scene